DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -194,35 +214,40 @@ private static void runStreamHlsAndDash(ConfigWrapper config) {
      * output is set to encode a video using the passed in preset.
      *
      * @param manager       The entry point of Azure Media resource management.
-     * @param resourceGroup The name of the resource group within the Azure subscription.
+     * @param resourceGroup The name of the resource group within the Azure
+     *                      subscription.
      * @param accountName   The Media Services account name.
      * @param transformName The name of the transform.
      * @return The transform found or created.
      */
-    private static Transform ensureTransformExists(MediaManager manager, String resourceGroup, String accountName,
-                                                   String transformName) {
+    private static Transform ensureTransformExists(MediaServicesManager manager, String resourceGroup,
+            String accountName,
+            String transformName) {
         Transform transform;
         try {
-            // Does a Transform already exist with the desired name? Assume that an existing Transform with the desired name
+            // Does a Transform already exist with the desired name? Assume that an existing
+            // Transform with the desired name
             // also uses the same recipe or Preset for processing content.
             transform = manager.transforms()
-                    .getAsync(resourceGroup, accountName, transformName)
-                    .toBlocking()
-                    .first();
-        } catch (NoSuchElementException nse) {
+                    .get(resourceGroup, accountName, transformName);
+        } catch (ManagementException nse) {
             // Media Services V3 throws an exception when not found.
             transform = null;
+        } catch (Exception e) {
+            System.out.println("failed" + e.toString());
+            transform = null;
         }
 
         if (transform == null) {
             List<TransformOutput> outputs = new ArrayList<>();
-            outputs.add(new TransformOutput().withPreset(new BuiltInStandardEncoderPreset().withPresetName(EncoderNamedPreset.ADAPTIVE_STREAMING)));
+            outputs.add(new TransformOutput().withPreset(
+                    new BuiltInStandardEncoderPreset().withPresetName(EncoderNamedPreset.ADAPTIVE_STREAMING)));
```

Comment:
Can we change this to use ContentAwareEncoding as default. I prefer that now to Adaptive Streaming. 
Adaptive streaming actually generates layers that are not needed, and upscales stuff.  So it is not putting our "best foot forward" for our samples.